scraibe/app/app.py

```diff
@@ -0,0 +1,72 @@
+"""
+Gradio App.
+--------------------------------
+
+This module provides an interface to transcribe audio files using the 
+Scraibe model. Users can either upload an audio file or record their speech 
+live for transcription. The application supports multiple languages and provides 
+options to specify the number of speakers and the language of the audio.
+
+Attributes:
+    LANGUAGES (list): A list of supported languages for transcription.
+
+Usage:
+    Run this script to start the Gradio web interface for audio transcription.
+    
+"""
+
+
+####
+# Gradio Interface
+####
+
+from threading import Thread
+
```

Comment:
Add Docstring which states the following imports are from internal Modules

Reply:
Done